.pre-commit-config.yaml

```diff
@@ -188,7 +188,8 @@ repos:
     name: MyPy, for Python 3.13
     additional_dependencies:
     - lxml  # dep of `--txt-report`, `--cobertura-xml-report` & `--html-report`
-    - pytest
+    - pytest # dep of `credential_plugins_test.py`& `importable_test.py`
```

Comment:
We run MyPy 3 times, against different Python versions — each of the invocations must have the same list of dependencies copied over, not just the Python 3.13 run.